### PR TITLE
Gela/remove oauth

### PR DESCRIPTION
**Pull request by Gela**
We have a one-off sample called "OAuth Redirect" which attempted to showcase how to use an external browser + redirects vs embedding a browser in the app window. This was experimental at best, very clunky, and only worked on a few platforms. We are removing this to avoid confusion.

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Features/FilterByDefinitionExpressionOrDisplayFilter/FilterByDefinitionExpressionOrDisplayFilter.cpp [hunk 1]
```diff
@@ -133,3 +139,34 @@ void FilterByDefinitionExpressionOrDisplayFilter::queryFeatureCountInCurrentExte
 
   m_featureTable->queryFeatureCount(parameters);
 }
+
```
Comment (tanneryould) [inline ArcGISRuntimeSDKQt_CppSamples/Features/FilterByDefinitionExpressionOrDisplayFilter/FilterByDefinitionExpressionOrDisplayFilter.cpp:142]: This looks like it was supposed to be included in a different PR?

**Review by tanneryould — APPROVED** [review]
Approved, but it looks like changes for "Filter by definition expression or display filter" were included in this PR.

File: ArcGISRuntimeSDKQt_CppSamples/Features/FilterByDefinitionExpressionOrDisplayFilter/FilterByDefinitionExpressionOrDisplayFilter.cpp (hunk 1 above)
Comment (Gela) [reply ArcGISRuntimeSDKQt_CppSamples/Features/FilterByDefinitionExpressionOrDisplayFilter/FilterByDefinitionExpressionOrDisplayFilter.cpp]: Yes, branching off an active branch on accident. Should be cleared up now
